### ISSUE TITLE
Sources no longer available

### ISSUE THREAD

**Issue by gronka** [Lucretia/vulkan-overlay#10]
The original links appear to have been moved. I've had trouble finding mirrors. And for the latest amdgpu-pro driver I can't find a link to amdgpu-pro-dkms on amd's site.

**Comment by Lucretia** [comment]
Latester [here](https://support.amd.com/en-us/kb-articles/Pages/AMD-Radeon-GPU-PRO-Linux-Beta-Driver%E2%80%93Release-Notes.aspx)

**Comment by gronka** [comment]
So dkms isn't necessary?

In make.conf should I set:
VIDEO_CARDS="amdgpu radeonsi"
I am using an r9 390

**Comment by Lucretia** [comment]
I'm surprised the first version even compiled for 4.6.x kernels and worked, tbh. Pure luck I think. 4.7.x and 4.8.x do not work with any of them so far.

**Comment by stobenski** [comment]
You can download [amdgpu-pro_16.30.3-306809.tar.xz](https://dl.dropboxusercontent.com/u/106367097/DRIVERS/amdgpu-pro_16.30.3-306809.tar.xz) from my Dropbox.

**Comment by ethan961** [comment]
I've been running 16.30.3-315407 for over a week now on kernel 4.6.5 without any obvious issues, just changed the version numbers in the ebuilds and it all worked, including the DKMS module. No later kernel worked.

**Comment by Lucretia** [comment]
@ethan961 it depends on what you're doing with it really. It'll probably be failing quietly in some cases. Currently the 4.6.x kernels lack the TTM LRU callbacks in the the ttm_bo_driver and this really should be added as some of the other code really makes use of it. By removing it, I'm not really sure what is happening.

**Comment by ethan961** [comment]
That's true, I don't do any dev work, just play Dota. I understand it's basically pure luck it works at all - what I meant was more that the 315407 build also seems to "work" in the same fashion without obvious errors. Did much actually change with that build to break things?

**Comment by Lucretia** [comment]
Yeah, a lot in the dkms module. I think I have found the relevant patches required to implement the mising ioctls in a custom kernel/libdem ebuild thus not requiring the dkms anymore.

**Comment by Lucretia** [comment]
Unfortunately the missing ioctls are not enough, there is more missing.
